gradle.properties

```diff
@@ -23,6 +23,7 @@ POM_SNAPSHOT_VERSION_NAME=2.1.0-SNAPSHOT
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xmx2048M
+systemProp.kotlin.daemon.jvm.options=-Xss8m
```

Comment:
Could you add a comment here explaining or linking to why we need to increase the stack size?

Just so we know where this comes from down the line.